src/simulation/navigation/simpleNav/simpleNav.rst

```diff
@@ -38,6 +38,9 @@ not come from a noisy sensor but from a ground-based estimation process.
     * - scEphemOutMsg
       - :ref:`EphemerisMsgPayload`
       - spacecraft ephemeris output msg
+    * - scEphemOutMsg
+      - :ref:`accelDataOutMsg`
```

Comment:
accelDataOutMsg or imuDataOutMsg just going based on the comment description?

Reply:
Wait, it looks like this is just gyro data, so shouldn't this be gyroDataOutMsg?

Reply:
The message was named accel Data from Scott. I just kept the name consistent